### PR TITLE
Add automated release with tags for Android & Web

### PR DESCRIPTION
**Pull request by jmartinesp**
I'll change the docs once we're sure that the jobs work as expected.

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -1,8 +1,63 @@
...
+          rustup target add aarch64-linux-android
+
+      - name: 🔨 Install Uniffi
+        uses: actions-rs/cargo@v1
+        with:
+          command: install
+          # keep in sync with uniffi dependency in Cargo.toml's
+          args: uniffi_bindgen --version ${{ env.UNIFFI_VERSION }}
+
+      - name: 🚀 Publish to Sonatype
```
Comment (andybalaam) [inline .github/workflows/publish.yml:49]: Should this be "Publish to Maven Central?"

File: update_version.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  SED_CMD="sed -i"
+fi
+
+VERSION=$1
+
+echo "Updating Rust"
+$SED_CMD "s/^version\s*=\s*\".*\"/version = \"$VERSION\"/g" bindings/wysiwyg-ffi/Cargo.toml
+$SED_CMD "s/^version\s*=\s*\".*\"/version = \"$VERSION\"/g" bindings/wysiwyg-wasm/Cargo.toml
+$SED_CMD "s/^version\s*=\s*\".*\"/version = \"$VERSION\"/g" crates/wysiwyg/Cargo.toml
+$SED_CMD "s/\"version\":\s*\".+\"/\"version\": \"$VERSION\"/g" bindings/wysiwyg-wasm/package.json
```
Comment (andybalaam) [inline update_version.sh:24]: This is part of Web, so should be moved down

**Review by andybalaam — APPROVED** [review]
Very minor comments - looks great - thanks!

File: .github/workflows/publish.yml (hunk 1 above)
Comment (jmartinesp) [reply .github/workflows/publish.yml:49]: Technically, we're publishing to [Sonatype](s01.oss.sonatype.org) that later synchronises with Maven Central.

**Pull request merged**